extensions/arc/src/common/utils.ts

```diff
@@ -282,8 +282,18 @@ export function convertToGibibyteString(value: string): string {
  * @param condition
  * @param message
  */
-export function throwUnless(condition: boolean, message?: string): asserts condition {
+export function throwUnless(condition: any, message?: string): asserts condition {
 	if (!condition) {
```

Comment:
if we do this 0 and '' will also cause it to throw, is that expected or you want to explicitly check undefined:
```
if (condition !== undefined)?
```

Reply:
Yes, that is intended and hence the change. If the caller wants to not do that they can always pass in a more specific condition. The previous implementation was unnecessarily restrictive, the new allows the caller full freedom to pass-in the condition checks as needed.